src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -413,7 +413,7 @@ def get_login_credentials(self, resource=None, client_id=None, subscription_id=N
             credential = self._create_credential(account, client_id=client_id)
             external_credentials = []
             for external_tenant in external_tenants:
-                external_credentials.append(self._create_credential(account, external_tenant, client_id=client_id))
+                external_credentials.append(self._create_credential(account, tenant_id=external_tenant))
```

Comment:
`_create_credential()` defines `tenant_id` as a keyword argument:

https://github.com/Azure/azure-cli/blob/0f28626d61c9a43379e28794b1fcf1d58dee20a1/src/azure-cli-core/azure/cli/core/_profile.py#L659

Calling `_create_credential()` should follow the same pattern.